src/style.css

```diff
@@ -2,0 +1,173 @@
+/* #region Reset Css */
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+font,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  outline: 0;
+  font-weight: inherit;
+  font-style: inherit;
+  font-size: 100%;
+  font-family: inherit;
+  vertical-align: baseline;
+}
+
+/* #endregion */
+
+/* #region Stars 0 */
+
+.stars--0 {
+  display: flex;
+}
```

Comment:
The classes `.stars--0`, `.stars--1`, `.stars--2`, `.stars--3`, `.stars--4`, `.stars--5` all have the same `display: flex;` property. You can simplify your code by creating a single `.stars` class with this property and applying it to all your stars elements.